src/components/FilterBar/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useEffect } from 'react'
+import Stack from '@mui/material/Stack'
+import MultiSelectCheckmarks from '../FilterControls/MultiSelect'
+
+import { Button } from '@mui/material'
+import {
+  MediaFilterActionTypes,
+  MediaFilterState,
+} from '../../types/interfaces/MediaFiltersReducer'
+import MediaTitleSearchInput from '../FilterControls/MediaTitleSearchInput'
+import MediaTypeRadioGroup from '../FilterControls/MediaTypeRadioGroup'
+
+const genreOptions = ['action', 'adventure', 'comedy']
+const yearsOptions = ['1981', '1987']
+
+interface FilterBarProps {
+  dispatch: ({ type, payload }: { type: any; payload?: any }) => void
+  state: MediaFilterState
```

Comment:
This is a medium-sized change, so don't feel like you need to listen to me, but I actually think `FilterBar` does not need state at all. It dispatches state updates via its children's `onChange` handlers, but those children are capable of knowing their own state, so we don't need it here.